release-notes/unreleased/IGAPP-550-add-georgian.yml

```diff
@@ -0,0 +1,7 @@
+issue_key: IGAPP-550
+show_in_stores: false
+platforms:
+  - web
+  - android
+  - ios
+en: Support added for georgian language and alphabet.
```

Comment:
I think it is only for web, isn't it?
Why no german release note?

Reply:
See answer in other PR, if `show_in_stores` is set to true, it is not shown to users and therefore no german version is needed. I don't want to show this to users as there is only one city (and that is not even live and a malteser one) that uses this new language and as a user I would really be confused why this language is not available in my city anyway.

Regarding only web: It is native as well, we just dont have an additonal font there. But the language (should) work on native as well, which it did not previously to this issue.